Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'sass-rails', '~> 5.0', '>= 5.0.6'
 gem 'turbolinks', '~> 5.0.0'
 gem 'uglifier', '~> 3.2.0'
 
-gem 'carrierwave', '~> 1.0.0'
+gem 'carrierwave', '~> 1.2.3'
```

Comment:
This is required because `carrierwave` uses a bunch of `attribute_changed?` hooks, which got deprecated in Rails 5.1. The updated version fixes the deprecation warnings, and carrierwave do a great job of following SemVer so the update shouldn't cause issues. (Score is [98%](https://dependabot.com/compatibility-score.html?dependency-name=carrierwave&package-manager=bundler&version-scheme=semver) on Dependabot!)